courtbot/states/CA/__init__.py

```diff
@@ -0,0 +1,123 @@
+
+import requests
+
+"""A package for retreiving court calendar data from Santa Clara County, California"""
+
+counties = [ 'santa clara' ]
+
+courts = [ 'santa clara' ]
+
+court_names = ["Civil",
+               "Probate",
+               "Family",
+               "Juvenile",
+               "Criminalhoj",
+               "CriminalPaloAlto",
+               "CriminalSouthCounty"]
+
+class Court:
+
+    def __init__(self, name, department):
+        self.__name = name
```

Comment:
I am using __init__ deliberately, because (as I understand it) a __init__ method in the class is called at class initialization time.

But see below. I think that your larger point is very valid.